torch/_inductor/triton_ops/autotune.py

```diff
@@ -675,29 +675,47 @@ def mm_autotune(get_io_bound_configs=False):
 def grid(xnumel, ynumel=None, znumel=None):
     """Helper function to compute triton grids"""
 
+    def get_grid_dim(numel, block_name, meta):
+        if numel is None:
+            return None
+        block = meta[block_name]
```

Comment:
nit: you don't need all of `meta`, you only need `meta[*BLOCK]` argument to this function

Reply:
Sure, changed that.